### PR TITLE
Update GraalVM to 21.1

### PR DESCRIPTION
**Pull request by Sgitario**

### CODE REVIEW

File: 301-quarkus-vertx-kafka/pom.xml [hunk 1]
```diff
@@ -17,14 +17,15 @@
...
     <dependencies>
-        <dependency>
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-opentelemetry</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-opentelemetry-exporter-jaeger</artifactId>
-        </dependency>
+        <!-- TODO: https://github.com/quarkusio/quarkus/issues/17234 -->
```
Comment (Sgitario) [inline 301-quarkus-vertx-kafka/pom.xml:20]: The same fix as in the module 300 done in https://github.com/quarkus-qe/beefy-scenarios/pull/206

**Review by rsvoboda — APPROVED** [review]

**Comment by Sgitario** [comment]
The issue https://github.com/quarkusio/quarkus/issues/17234 has been fixed. I've already updated this PR.

**Comment by rsvoboda** [comment]
Windows run fails with 
```
Error:  Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 32.253 s <<< FAILURE! - in io.quarkus.qe.many.extensions.ManyExtensionsResourceTest
Error:  testHelloEndpoint  Time elapsed: 0.013 s  <<< ERROR!
java.lang.RuntimeException: 
java.lang.RuntimeException: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
Error: ]: Build step io.quarkus.kafka.client.deployment.DevServicesKafkaProcessor#startKafkaDevService threw an exception: java.lang.IllegalStateException: Could not find a valid Docker environment. Please see logs and check configuration
```

**Comment by Sgitario** [comment]
> Windows run fails with
> 
> ```
> Error:  Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 32.253 s <<< FAILURE! - in io.quarkus.qe.many.extensions.ManyExtensionsResourceTest
> Error:  testHelloEndpoint  Time elapsed: 0.013 s  <<< ERROR!
> java.lang.RuntimeException: 
> java.lang.RuntimeException: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
> Error: ]:
>  Build step io.quarkus.kafka.client.deployment.DevServicesKafkaProcessor#startKafkaDevService threw an exception: 
>  java.lang.IllegalStateException: Could not find a valid Docker environment. Please see logs and check configuration
> ```

This is unrelated to this PR, but do you want me to exclude this module from the windows build as it now needs Docker? (as Kafka is part of the dev services now)

**Comment by rsvoboda** [comment]
> do you want me to exclude this module from the windows build as it now needs Docker? (as Kafka is part of the dev services now)

yes please, new PR or additional commit to this PR

**Comment by Sgitario** [comment]
> > do you want me to exclude this module from the windows build as it now needs Docker? (as Kafka is part of the dev services now)
> 
> yes please, new PR or additional commit to this PR

Added an additional commit to this PR and also reverted the workaround I did as the issue has been already fixed in upstream.

**Pull request merged**